src/CameraRoll.ts

```diff
@@ -238,3 +239,51 @@ export class CameraRoll {
     return RNCCameraRoll.getPhotoByInternalID(internalID, conversionOption);
   }
 }
+
+const initialState: PhotoIdentifiersPage = {
+  edges: [],
+  page_info: {
+    end_cursor: '',
+    has_next_page: false,
+    start_cursor: '',
+  },
+};
+
+const defaultConfig: GetPhotosParams = {
+  first: 20,
+  groupTypes: 'All',
+};
+
+type UseCameraRollResult = [
+  PhotoIdentifiersPage,
+  (config?: GetPhotosParams) => Promise<void>,
+  (tag: string, type?: 'photo' | 'video' | 'auto') => Promise<void>,
+];
+
+export function useCameraRoll(): UseCameraRollResult {
+  const [photos, setPhotos] = useState<PhotoIdentifiersPage>(initialState);
+
+  async function getPhotos(
+    config: GetPhotosParams = defaultConfig,
+  ): Promise<void> {
+    try {
+      const result = await CameraRoll.getPhotos(config);
+      setPhotos(result);
+    } catch (error) {
+      if (__DEV__) console.log('[useCameraRoll] Error getting photos: ', error);
+    }
+  }
+
+  async function saveToCameraRoll(
+    ...args: Parameters<typeof CameraRoll.saveToCameraRoll>
+  ): Promise<void> {
+    try {
+      await CameraRoll.saveToCameraRoll(...args);
```

Comment:
Could you change this to use the `save` function and not the deprecated `saveToCameraRoll` instead?

Reply:
done

Reply:
done